params/types/parity/parity.go

```diff
@@ -106,8 +106,10 @@ type ParityChainSpec struct {
 		EIP1344Transition         *ParityU64 `json:"eip1344Transition,omitempty"`
 		EIP1884Transition         *ParityU64 `json:"eip1884Transition,omitempty"`
 		EIP2028Transition         *ParityU64 `json:"eip2028Transition,omitempty"`
+		EIP2315Transition         *ParityU64 `json:"eip2315Transition,omitempty"`
 		EIP2537Transition         *ParityU64 `json:"eip2537Transition,omitempty"`
 		EIP1706Transition         *ParityU64 `json:"-"` // FIXME, when and if i'm implemented in Parity
+		EIP2929Transition         *ParityU64 `json:"-"` // FIXME, when and if i'm implemented in Parity
```

Comment:
We have a test case on forks (TestEquivalent_Features) that fails on this transition. The thing is that OpenEthereum is not going to implement this EIP as they are out of resources at this moment. How can we handle this test?

Reply:
I'm guessing `func (spec *ParityChainSpec) SetEIP2929Transition(n *uint64) error {` needs to actually set `EIP2929Transition`, in the same way as EIP2315 is handled for Parity.

Reply:
Resolved with https://github.com/etclabscore/core-geth/pull/246/commits/f3aff0ec9c2ec7342973e19f739c881c226bd867

Reply:
This resolves our test but I though of leaving it unsupported based on the comment [here](https://docs.google.com/spreadsheets/d/1BomvS0hjc88eTfx1b8Ufa6KYS3vMEb2c8TQ5HJWx2lc/edit#gid=408811124) that says that they are not going to implement this.

Reply:
Ah, roger that. I've reverted that commit and just modified the tests to skip Parity :: [Berlin|YOLO] tests. See https://github.com/etclabscore/core-geth/pull/246/commits/186e3cb59becdd812fa4c57da6ec3e4458e899dc and subsequent.